docs/security/tokens/using-tokens.md

```diff
@@ -0,0 +1,237 @@
+       You'll need to remember this if you want to re-use this profile in subsequent sessions.
+
+#### Requesting access tokens
+
+!!! note
+    You must first [register a new profile](#registering-an-oidc-profile).
+
+1. Request a token using the client profile that you used with `oidc-gen`:
+	
+        :::console
+        docker exec -it my-agent oidc-token --aud="<SERVER AUDIENCE>" <CLIENT PROFILE>
+
+
+    For tokens used against an HTCondor-CE, set `<SERVER AUDIENCE>` to `<CE FQDN>:<CE PORT>`.
+
+1. Copy the output of `oidc-token` into a file on the host where you need bearer token authentication, e.g. an HTCondor
+   or XRootD client.
+
+#### Reloading an OIDC profile
+
+!!! note
+    Required after restarting the running container. You must have an existing [registered profile](#registering-an-oidc-profile).
+
+1. If your existing container is not already running, start it:
+
+        :::console
+        docker start my-agent
+
+1. Reload profile:
+
+        :::console
+        docker exec -it my-agent oidc-add <CLIENT PROFILE>
+
+
+1. Enter password used to encrypt your `<CLIENT PROFILE>` created during profile registration.
+
+
+### Using an RPM installation
+
+#### Registering an OIDC profile
+
+1. Start the agent and add the appropriate variables to your environment:
+
+        :::console
+        eval `oidc-agent`
+
+1. Generate a local client profile and follow the prompts:
+
+        :::console
+        oidc-gen -w device <CLIENT PROFILE>
+
+    1. Specify an OIDC provider such as CILogon or an IAM instance as the client issuer.
+       For example, if you are requesting tokens from the WLCG IAM instance:
+
+            Issuer [https://iam-test.indigo-datacloud.eu/]: https://wlcg.cloud.cnaf.infn.it/
+
+    1. Request scopes for the capabilities that you need based on the type of tokens that your provider issues:
+
+        | **Capability**   | **SciTokens Scope** | **WLCG Scope**                                 |
+        |:-----------------|---------------------|------------------------------------------------|
+        | HTCondor `READ`  | `condor:/READ`      | `compute.read`                                 |
+        | HTCondor `WRITE` | `condor:WRITE`      | `compute.modify compute.cancel compute.create` |
+        | XRootD read      | `read:<PATH>`       | `storage.read:<PATH>`                          |
+        | XRootD write     | `write:<PATH>`      | `storage.modify:<PATH>`                        |
+
+        Replacing `<PATH>` with a path to the storage location that the bearer should be authorized to access.
+        If you are requesting WLCG tokens, you will need to also add the `wlcg` and `offline_access` scopes.
+        For example, to request HTCondor `READ` and `WRITE` access from an OIDC provider issuing WLCG tokens,
+        specify the following scopes:
+
+            This issuer supports the following scopes: openid profile email address phone offline_access wlcg iam wlcg.groups
```

Comment:
Same comment as above for distinguishing the prompt from the user input.

Reply:
Done